packages/api-headless-cms/src/handler/plugins/modelFields/datetime.ts

```diff
@@ -10,7 +10,7 @@ enum DATE_TYPE {
     TIME = "time"
 }
 
-function createValidation(validation, format) {
+function createValidation(validation, format, checkLength = true) {
```

Comment:
Let's try switching places:
![image](https://user-images.githubusercontent.com/5121148/81810393-28da2380-9523-11ea-9f7b-f948dfaadf1c.png)

No sense in executing custom validation if the data itself is in invalid format.